src/app/assets/styles/_color_variables.scss

```diff
@@ -14,5 +14,6 @@
 
 // TODO(floreks): Find appropriate place for global color definitions.
 $primary: #326DE6;
-$line: #AAAAAA;
-$subline: #888888;
+$lightGray: #AAAAAA;
```

Comment:
Actually, previous names were better :) They were about the function of the color, not implementation. Imagine that we introduce themes to our product, then `$lightGray` would become, e.g., green, which makes little sense :) 

If `$line` name was bad, try to find better, but functional, name. "Secondary", "title", "caption", "heading" are examples of functional names. 

@floreks